src/ape/cli/options.py

```diff
@@ -99,3 +103,32 @@ def account_option_that_prompts_when_not_given():
         type=AccountAliasPromptChoice(),
         callback=_account_callback,
     )
+
+
+def _load_contracts(ctx, param, value) -> Optional[Union[ContractType, List[ContractType]]]:
+    if not value:
+        return None
+
+    if len(project.contracts) == 0:
+        raise ContractError("Project has no contracts.")
+
+    is_multiple = isinstance(value, (tuple, list))
+
+    def create_contract(contract_name: str) -> ContractType:
+        if contract_name not in project.contracts:
+            raise ContractError(f"No contract named '{value}'")
+
+        return project.contracts[contract_name]
+
+    return [create_contract(c) for c in value] if is_multiple else create_contract(value)
```

Comment:
I think if the user passes in `multiuple=True`, they are expecting a list. Otherwise, they are not.

Like, if the user did not know they _could_ possible `multiple=True` for some reason, it would be a little surprising to get a list... Like in the `How To Verify It` first example.

Reply:
gotcha, okay this is fine then